src/containers/ColumnHeader/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+
+import useColumnHeaderStyles from './columnHeader.styles';
+
+interface IColumnHeader {
+  name: string;
+}
+
+const ColumnHeader: FC<IColumnHeader> = ({ name }) => {
+  const styles = useColumnHeaderStyles();
+
+  return <div className={styles.header}>{name}</div>;
```

Comment:
Maybe you can use some default material components (Typography for example) and use h2 or similar components